package.json

```
@@ -148,24 +147,23 @@
     "punycode": "^2.1.1",
     "qrcode-generator": "1.4.1",
     "ramda": "^0.24.1",
-    "react": "^15.6.2",
+    "react": "^16.0.0",
     "react-dnd": "^3.0.2",
     "react-dnd-html5-backend": "^7.4.4",
-    "react-dom": "^15.6.2",
+    "react-dom": "^16.0.0",
```

We could put a newer version here to make sure we don't ever see a downgrade:

```suggestion
    "react-dom": "^16.12.0",
```

Yarn resolved this dependency to `16.12.0` so this seems safe enough.